pylint_odoo/misc.py

```diff
@@ -112,14 +113,27 @@ def get_manifest_file(self, node_file):
 
     def set_ext_files(self):
         """Create `self.ext_files` dictionary with {extension_file: [files]}
+            and exclude files using --ignore and --ignore-patterns parameters
         """
         self.ext_files = {}
         for root, _, filenames in os.walk(self.module_path, followlinks=True):
             for filename in filenames:
+                find = True
                 fext = os.path.splitext(filename)[1].lower()
                 fname_rel = os.path.relpath(
                     os.path.join(root, filename), self.module_path)
-                self.ext_files.setdefault(fext, []).append(fname_rel)
+                name = os.path.join(self.module_path, fname_rel)
+                # If the file is within black_list_re is ignored
+                if _basename_in_blacklist_re(name,
+                                             self.linter.config.black_list_re):
+                    continue
+                # If the file is within ignores is ignored
+                for ignore in self.linter.config.black_list:
+                    if ignore in name:
+                        find = False
```

Comment:
What means find variable?
(Because I don't understand why we have a `find = False` when is really find it)